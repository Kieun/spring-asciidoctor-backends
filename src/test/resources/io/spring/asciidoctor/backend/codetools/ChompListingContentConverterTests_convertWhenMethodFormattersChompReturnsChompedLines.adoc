= Test Document
:chomp: formatters

[source,java]
----
public class Example {

	public void method() {
		// @formatter:off
		new StringBuilder();
		// @formatter:on
	}

}
----
