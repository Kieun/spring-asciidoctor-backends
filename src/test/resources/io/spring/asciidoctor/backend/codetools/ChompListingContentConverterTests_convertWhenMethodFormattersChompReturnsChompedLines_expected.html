public class Example {

	public void method() {
		new StringBuilder();
	}

}